postgres/server/init.sls

```diff
@@ -56,7 +56,7 @@ postgresql-cluster-prepared:
     - name: {{ postgres.prepare_cluster.command }}
     - cwd: /
     - runas: {{ postgres.prepare_cluster.user }}
-    - env: {{ postgres.prepare_cluster.env|default({}) }}
+    - env: {{ postgres.prepare_cluster.env|default({}) or None }}
```

Comment:
I suggest to edit `defaults.yaml` instead, like this:
```yaml
# defaults.yaml
postgres:
  ...
  prepare_cluster:
    command: initdb --pgdata=/var/lib/pgsql/data
    test: test -f /var/lib/pgsql/data/PG_VERSION
    user: postgres
    env: Null  # Be aware that there's no ``None`` value exists in YAML
```

And just use `- env: {{ postgres.prepare_cluster.env }}` over here.